dotnet-authserver/src/TeacherIdentity.AuthServer/Journeys/LegacyTrnJourney.cs

```diff
@@ -3,45 +3,200 @@
 
 namespace TeacherIdentity.AuthServer.Journeys;
 
-public class LegacyTrnJourney : TrnLookupJourneyBase
+public class LegacyTrnJourney : SignInJourney
```

Comment:
I did away with the `TrnLookupJourneyBase` initially proposed; it didn't end up being super useful.